library/build.go

```diff
@@ -158,6 +159,7 @@ func (b *Build) Environment(workspace, channel string) map[string]string {
 		envs["VELA_BUILD_TARGET"] = target
 		envs["VELA_DEPLOYMENT"] = target
 		envs["BUILD_TARGET"] = target
+		envs["VELA_DEPLOYMENT_NUMBER"] = ToString(b.GetDeploymentID())
```

Comment:
One of the benefits of storing our own deployment record is we can create our own `Environment` function for the deployment type. Might be worth considering here. At bare minimum, this should be `VELA_DEPLOYMENT_ID`, which is arguably not worth including 😅

Reply:
that's fair - i like that